core/cmd/local_client.go

```diff
@@ -188,7 +185,10 @@ func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
 					Name:   "version",
 					Usage:  "Display the current database version.",
 					Action: client.VersionDatabase,
-					Flags:  []cli.Flag{},
+					Before: func(ctx *clipkg.Context) error {
```

Comment:
note, i only added the validation to the public facing cmds. do we want them on the commands?